wp-dependency-installer.php

```diff
@@ -84,15 +84,14 @@ public function load_hooks() {
 		 * Register wp-dependencies.json
 		 *
 		 * @param string $plugin_path Path to plugin or theme calling the framework.
+		 * @param array  $config (optional) JSON config as array.
 		 */
-		public function run( $plugin_path ) {
+		public function run( $plugin_path, $config = [] ) {
```

Comment:
I do like the better flow here but I don't think passing a `$config` here is worth the added complexity. Can you remove this part from the PR?